setup.py

```diff
@@ -14,6 +14,8 @@
 
 current_date = datetime.now().strftime("%Y%m%d")
 
+PY3_9_HEXCODE = "0x03090000"
```

Comment:
this strange AF to me as a flag

Reply:
lol yea welcome to the undocumented world of python